src/pages/home/report/ReportActionsList.js

```diff
@@ -177,12 +177,13 @@ class ReportActionsList extends React.Component {
                         // Make sure the oldest report action loaded is not the first. This is so we do not show the
                         // skeleton view above the created action in a newly generated optimistic chat or one with not
                         // that many comments.
+                        // We will show non-animating skeleton UI if we are offline and chat history is not loaded in ONYX.
```

Comment:
Comment does not explain "why" only the "what". Also let's try to be more consistent about capitalization. In other comments we have "Onyx" and this one "ONYX".

```suggestion
                        // We will show a static skeleton UI if we are offline and chat history is not loaded in Onyx because...
```